lms/templates/courseware/course_about.html

```diff
@@ -240,22 +240,21 @@ <h1>
           % endif
             ## We plan to ditch end_date (which is not stored in course metadata),
             ## but for backwards compatibility, show about/end_date blob if it exists.
-            % if get_course_about_section(request, course, "end_date") or course.end:
-                <%
-                    course_end_date = course.end
-                %>
-
+            <%
+               course_end_date = get_course_about_section(request, course, "end_date") or course.end
```

Comment:
I *think* `get_course_about_section(request, course, "end_date")` is a deprecated idiom.

Reply:
It still exists, but I don't `end_date` takes "self paced" courses into account, it's not very useful...`course.end` on the other hand does understand the concept of "self paced".